.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       - image: circleci/php:7.1-node-browsers
```

Comment:
Good catch, we should update it here, although we don't have a way to validate it since we don't have circle CI set up for the project. This configuration will likely go away when we address #21 in favor of a Github action, but we can updated it now to keep it in sync.